Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -107,10 +115,20 @@ void MainWindow::CreateComponents()
   m_controllers_window = new ControllersWindow(this);
   m_settings_window = new SettingsWindow(this);
   m_hotkey_window = new MappingWindow(this, 0);
-
+#if defined(HAVE_XRANDR) && HAVE_XRANDR
+  m_graphics_window = new GraphicsWindow(
+      this,
+      new X11Utils::XRRConfiguration(
+          static_cast<Display*>(QGuiApplication::platformNativeInterface()->nativeResourceForWindow(
```

Comment:
``XDisplayFromHandle`` seems to be ![unimplemented](https://github.com/dolphin-emu/dolphin/search?q=XDisplayFromHandle) (except for Wx)